tests/test_optimizer.py

```diff
@@ -170,3 +173,108 @@ def test_load_state_from_peers():
     assert avgr1.local_epoch == 1337
     assert torch.all(model1.weight == 42).item()
     assert np.allclose(avgr1.optimizer.param_groups[0]["lr"], 0.1 / 1337)
+
+
+@pytest.mark.forked
+def test_progress_tracker():
+    # note to a curious reader: no, you cannot reduce the timings without compromising realism or stability
+    prefix = "my_exp"
+    target_batch_size = 256
+    dht_root = hivemind.DHT(start=True)
+    barrier = mp.Barrier(parties=5)
+    delayed_start_evt = mp.Event()
+    finished_evt = mp.Event()
+    emas = mp.Array(ctypes.c_double, 5)
+
+    def worker(index: int, batch_size: int, period: float, **kwargs):
```

Comment:
```suggestion
    def run_worker(index: int, batch_size: int, period: float, **kwargs):
```

nit: This makes it clear that this is a function, not an object.